TweekApiSmokeTestDockerfile

```diff
@@ -0,0 +1,40 @@
+# ---- *.csproj FILES ----
+FROM microsoft/aspnetcore-build:2.0.0
+
+ENV RUBY_MAJOR 2.4
+ENV RUBY_VERSION 2.4.2
+ENV RUBY_DOWNLOAD_SHA256 748a8980d30141bd1a4124e11745bb105b436fb1890826e0d2b9ea31af27f735
```

Comment:
where is it used?

Reply:
Are you referring to `ruby`?

Reply:
Do we have to run Glue inside the e2e/integration/smoke tests image, or can we run it in a stand alone container (or aside the proxy) that collect the reports after the tests finish running?

Reply:
We can - but it will make the things a lot more complicated - you will have to somehow call this container at the end of each one of the tests (API maybe?). We can add this as improvement later.